README.md

```diff
@@ -10,28 +10,35 @@ will grant a set of temporary credential
 $ curl localhost:8080/v1/credentials
 {
   "credentials": {
-    "clientId": "localhost.localdomain",
+    "clientId": "project/testing-host-secrets/host/localhost.localdomain",
     "accessToken": "<snip>",
-    "certificate": "{\"version\":1,\"scopes\":[\"assume:project:testing-host-secrets:host:localdomain.localhost\",\"assume:project:testing-host-secrets:host:localhost6\",\"assume:project:testing-host-secrets:host:localdomain6.localhost6\"],\"start\":1486575908837,\"expiry\":1486575908837,\"seed\":\"<snip>\",\"signature\":\"<snip>\",\"issuer\":\"<snip>\"}"
+    "certificate": "{\"version\":1,\"scopes\":[\"assume:project:testing-host-secrets:host:localdomain.localhost\"],\"start\":1486575908837,\"expiry\":1486575908837,\"seed\":\"<snip>\",\"signature\":\"<snip>\",\"issuer\":\"<snip>\"}"
   }
 }
 ```
 
-Note that this generates a scope for all DNS names which this IP has when doing a reverse lookup.  In this example case, that is:
+The credentials generated give a single scope.  This scope is the concatenation
+of the `TASKCLUSTER_SCOPE_BASE` configuration value and the reversed domain
+name.  The client id for the temporary credential is the concatenation of the
+`TASKCLUSTER_CLIENT_ID_BASE` configuration value and the domain name.
 
-* `assume:project:testing-host-secrets:host:localdomain.localhost`
-* `assume:project:testing-host-secrets:host:localhost6`
-* `assume:project:testing-host-secrets:host:localdomain6.localhost6`
+For example, if `TASKCLUSTER_SCOPE_BASE` was set to `assume:project:test:host:`
+and `TASKCLUSTER_CLIENT_ID_BASE` was set to `project/test/host/` then the scope
+generated for a host with a hostname of `a.test.machine.com` would be
+`assume:project:test:host:com.machine.test.a` and the client id would be
+`project/test/host/com.machine.test.a`.
 
 The role specified in these scopes (`project:testing-host-secrets:host:*`) can
 be given other roles to expand on what they can do.  This includes storing secrets
-using the taskcluster-secrets service.
+using the taskcluster-secrets service.  This credential is used for bootstraping
+and not for generating the final credential to be used by the service.
```

Comment:
What does this last sentence mean?  I had been envisioning upgrading taskcluster-worker to consult this service periodically to get a "fresh" temporary credential, and using that credential both to access worker secrets like the TLS certificate and to claim work from the queue.